backends-velox/src/main/scala/org/apache/gluten/execution/ColumnarPartialProjectExec.scala

```diff
@@ -183,9 +181,8 @@ case class ColumnarPartialProjectExec(original: ProjectExec, child: SparkPlan)(
             } else {
               val start = System.currentTimeMillis()
               val childData = ColumnarBatches.select(batch, projectIndexInChild.toArray)
```

Comment:
Is there possibility that `projectIndexInChild` includes duplicated indexes? If yes can we have a test case for that? Thanks.

Reply:
No, it won't be duplicated, I have checked exists status when I append to it.